stubs/xxhash/xxhash.pyi

```diff
@@ -10,30 +11,39 @@ VERSION: str
 XXHASH_VERSION: str
 
 class _IntDigestHash(_Hash):
-    seed: int
-    digestsize: int
-    def __init__(self, __string: ReadableBuffer = ..., seed: int = ...) -> None: ...
+    @property
+    def seed(self) -> int: ...
+    @property
+    def digestsize(self) -> int: ...
+    def __init__(self, input: ReadableBuffer = ..., seed: SupportsIndex = ...) -> None: ...
     def intdigest(self) -> int: ...
     def reset(self) -> None: ...
 
 # python-xxhash v2.0.0 does not support the string or usedforsecurity kwargs
+@final
 class xxh32(_IntDigestHash): ...
+
+@final
 class xxh64(_IntDigestHash): ...
+
+@final
 class xxh3_64(_IntDigestHash): ...
+
+@final
 class xxh3_128(_IntDigestHash): ...
 
-def xxh32_digest(input: ReadableBuffer = ..., seed: int = ...) -> bytes: ...
-def xxh32_intdigest(input: ReadableBuffer = ..., seed: int = ...) -> int: ...
-def xxh32_hexdigest(input: ReadableBuffer = ..., seed: int = ...) -> str: ...
-def xxh64_digest(input: ReadableBuffer = ..., seed: int = ...) -> bytes: ...
-def xxh64_intdigest(input: ReadableBuffer = ..., seed: int = ...) -> int: ...
-def xxh64_hexdigest(input: ReadableBuffer = ..., seed: int = ...) -> str: ...
-def xxh3_64_digest(input: ReadableBuffer = ..., seed: int = ...) -> bytes: ...
-def xxh3_64_intdigest(input: ReadableBuffer = ..., seed: int = ...) -> int: ...
-def xxh3_64_hexdigest(input: ReadableBuffer = ..., seed: int = ...) -> str: ...
-def xxh3_128_digest(input: ReadableBuffer = ..., seed: int = ...) -> bytes: ...
-def xxh3_128_intdigest(input: ReadableBuffer = ..., seed: int = ...) -> int: ...
-def xxh3_128_hexdigest(input: ReadableBuffer = ..., seed: int = ...) -> str: ...
+def xxh32_digest(input: ReadableBuffer, seed: SupportsIndex = ...) -> bytes: ...
+def xxh32_intdigest(input: ReadableBuffer, seed: SupportsIndex = ...) -> int: ...
+def xxh32_hexdigest(input: ReadableBuffer, seed: SupportsIndex = ...) -> str: ...
+def xxh64_digest(input: ReadableBuffer, seed: SupportsIndex = ...) -> bytes: ...
+def xxh64_intdigest(input: ReadableBuffer, seed: SupportsIndex = ...) -> int: ...
+def xxh64_hexdigest(input: ReadableBuffer, seed: SupportsIndex = ...) -> str: ...
+def xxh3_64_digest(input: ReadableBuffer, seed: SupportsIndex = ...) -> bytes: ...
+def xxh3_64_intdigest(input: ReadableBuffer, seed: SupportsIndex = ...) -> int: ...
+def xxh3_64_hexdigest(input: ReadableBuffer, seed: SupportsIndex = ...) -> str: ...
+def xxh3_128_digest(input: ReadableBuffer, seed: SupportsIndex = ...) -> bytes: ...
+def xxh3_128_intdigest(input: ReadableBuffer, seed: SupportsIndex = ...) -> int: ...
+def xxh3_128_hexdigest(input: ReadableBuffer, seed: SupportsIndex = ...) -> str: ...
```

Comment:
Apparently the input can be a string. We can fix that in this PR or in a separate PR.

Reply:
No need, done as of https://github.com/python/typeshed/pull/6031/commits/fe55b5639f820fa3095c5564cdac0aed9ec2b745.